src/jit/regalloc.cpp

```diff
@@ -293,13 +293,13 @@ regMaskTP               Compiler::genReturnRegForTree(GenTreePtr tree)
 {
     var_types type = tree->TypeGet();
 
-#ifdef _TARGET_ARM_
+#ifdef FEATURE_HFA
```

Comment:
This #ifdef can be removed provided that IsHfa(GenTreePtr) is implemented to return false for non-hfa targets.